src/cli.zig

```diff
@@ -327,6 +327,8 @@ pub const Command = struct {
         coverage: TestCommand.CodeCoverageOptions = .{},
         test_filter_pattern: ?[]const u8 = null,
         test_filter_regex: ?*RegularExpression = null,
+        line_filter_files: std.ArrayListUnmanaged([]const u8) = .{},
+        line_filter_lines: std.ArrayListUnmanaged(u32) = .{},
```

Comment:
this is now even worse - it not only is still a duplicate field, its now got 2 (so in total 3 vars for what only should be 1 at most)